src/sentry/mediators/external_requests/select_requester.py

```diff
@@ -65,11 +65,26 @@ def _make_request(self):
         if not self._validate_response(response):
             raise APIError()
 
-        return response
+        return self._format_response(response)
 
     def _validate_response(self, resp):
         return validate(instance=resp, schema_type='select')
 
+    def _format_response(self, resp):
+        # the UI expects the following form:
+        # choices: [[label, value]]
+        # default: [label, value]
+        response = {}
+        choices = []
+
+        for option in resp:
+            choices.append([option['label'], option['value']])
+            if option.get('default'):
+                response['default'] = option['default']
```

Comment:
Don't we want `response['default'] = option` here? Otherwise it'd be setting it to `true`.

Reply:
We should also add a test for this `default` scenario.

Reply:
derp yeah